README.md

```diff
@@ -2,29 +2,137 @@
 [![Build Status](https://travis-ci.org/EmbedITCZ/jbehave-support.svg?branch=master)](https://travis-ci.org/EmbedITCZ/jbehave-support)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/1e5c39dfaa6240b8b448d0df114c0d8e)](https://www.codacy.com/app/jbehavesupport/jbehave-support?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=EmbedITCZ/jbehave-support&amp;utm_campaign=Badge_Grade)
 [![Gitter](https://badges.gitter.im/jbehave-support/community.svg)](https://gitter.im/jbehave-support/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
 # jbehave-support
+
 Light extension to [JBehave](https://jbehave.org).
 
-## how to build
-- checkout project to local disk
-- build project with Maven
-```bash
-mvn clean install
+## Contents
+
+1. [Simple use case](#simple-use-case)
+    1. [Add to Java project as a Maven dependency](#add-to-java-project-as-a-maven-dependency)
+    2. [Create a TestConfig configuration file](#create-a-testconfig-configuration-file)
+    3. [Create a ui-mapping file](#create-a-ui-mapping-file)
+    4. [Write your story](#write-your-story)
+    5. [Write your story class](#write-your-story-class)
+2. [Modules](#modules)
+    
+## Simple use case
+
+To show you how to set up a jbehave-support project, I am going to make a test that Google searches `EmbedITCZ jbehave-support` and checks the result. To learn more about this example check out [Web-testing.md](jbehave-support-core/docs/Web-testing.md).
+
+### Add to Java project as a Maven dependency
+
+To add jbehave-support to a java project, just add [this](https://mvnrepository.com/artifact/org.jbehavesupport/jbehave-support-core) dependency to your pom.xml.
+```
+<dependency>
+    <groupId>org.jbehavesupport</groupId>
+    <artifactId>jbehave-support-core</artifactId>
+    <version>1.0.8</version>
```

Comment:
Not sure about what to do with the version since it will change and github does not support any placeholders for it... Maybe omit it alltogether?